web/src/flow/stages/captcha/CaptchaStage.ts

```diff
@@ -43,19 +60,30 @@ export class CaptchaStage extends BaseStage<CaptchaChallenge, CaptchaChallengeRe
         ];
     }
 
-    handlers = [this.handleGReCaptcha, this.handleHCaptcha, this.handleTurnstile];
+    handlers: CaptchaHandler[] = [
+        {
+            name: "grecaptcha",
+            interactive: this.renderGReCaptchaFrame,
+            execute: this.executeGReCaptcha,
+        },
+        {
+            name: "hcaptcha",
+            interactive: this.renderHCaptchaFrame,
+            execute: this.executeHCaptcha,
+        },
+        {
+            name: "turnstile",
+            interactive: this.renderTurnstileFrame,
+            execute: this.executeTurnstile,
+        },
+    ];
 
     @state()
     error?: string;
 
-    @state()
```

Comment:
These objects were instantiated at build time and _never changed_.  There's no reason for them to be stateful or reactive.